src/utilities/xml/XMLValidator.cpp

```diff
@@ -477,8 +477,8 @@ XMLValidator XMLValidator::gbxmlValidator() {
     LOG_AND_THROW("Failed to create a temporary directory for extracting the embedded path");
   }
   const bool quiet = true;
-  ::openstudio::embedded_files::extractFile(":/xml/resources/GreenBuildingXML_Ver6.01.xsd", openstudio::toString(tmpDir), quiet);
-  return XMLValidator(tmpDir / "GreenBuildingXML_Ver6.01.xsd");
+  ::openstudio::embedded_files::extractFile(":/xml/resources/GreenBuildingXML_Ver7.03.xsd", openstudio::toString(tmpDir), quiet);
```

Comment:
Maybe we should use an enum (or a string with validation... ) and modifying the gbxmlValidator signature to take en optional gbxmlVersion parameter?

```c++
enum class GbxmlVersion { Ver601, Ver703 };
XMLValidator gbxmlValidator(GbxmlVersion gbxmlVersion = GbxmlVersion::Ver601);
```

The source code in `src/gbxml` needs to be modified to match as well.

Reply:
https://github.com/NREL/OpenStudio/blob/950172bc1247f6059a48d5ec1c314fab283a18de/src/gbxml/ForwardTranslator.cpp#L185-L191